kraken-std/.kraken.lock

```diff
@@ -5,7 +5,7 @@ requirements = [
 pythonpath = [
     "build-support",
 ]
-interpreter_constraint = ">=3.10,<3.11"
+interpreter_constraint = ">=3.7"
```

Comment:
Can you upgrade your Kraken-wrapper version? :)

Unfortunately the interpreter constraint is not currently informed by the dependencies (as `kraken-core` now requires 3.10 since 0.25.0), but it's put there by `krakenw` if it's not specified in the `buildscript()` call.